index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Fork Me! FCC: Test Suite Template</title>
```

Comment:
Change required:
A proper page title is suggested. It's better to bring to the user the general idea of the web page, and its functionality. 👍